### PR TITLE
Ja3 Fingerprinting

### PR DESCRIPTION
**Pull request by xsedla1o**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,8 +170,8 @@ List of unirec fields exported together with basic flow fields on interface by R
 | RTSP_RESPONSE_SERVER         | string | RTSP response server field  |
 | RTSP_RESPONSE_CONTENT_TYPE   | string | RTSP response content type  |
 
-### HTTPS
-List of unirec fields exported together with basic flow fields on interface by HTTPS plugin.
+### TLS
+List of unirec fields exported together with basic flow fields on interface by TLS plugin.
 
 | UniRec field        | Type   | Description                  |
 |:-------------------:|:------:|:----------------------------:|
```
Comment (cejkato2) [inline README.md:177]: The plugin itself is renamed but output fields are the same?

**Review by hynekkar — COMMENTED** [review]
Have you tried tests? The test reference file content did not change.

File: README.md (hunk 1 above)
Comment (xsedla1o) [reply README.md:177]: My mistake, README wasn't up to date. The fields have changed, and the README reflects that now.

**Comment by xsedla1o** [comment]
Good point, the tests are now up to date with the new output format.

**Comment by hynekkar** [comment]
The JA3 output of the plugin is not correct. It does not skip the google GREASE values in TLS Client Hello (grease described here: https://tools.ietf.org/html/draft-davidben-tls-grease-01). These values MUST be skipped to ensure proper function in TLS fingerprinting. 

I found the issue with traffic captured from my browser (pcap included bellow). The output of the plugin was: 
```
771,2570-4865-4866-4867-49195-49199-49196-49200-52393-52392-49171-49172-156-157-47-53,10794-0-23-65281-10-11-35-16-5-13-18-51-45-43-27-60138-21,6682-29-23-24,0

771,51914-4865-4866-4867-49195-49199-49196-49200-52393-52392-49171-49172-156-157-47-53,47802-0-23-65281-10-11-35-16-5-13-18-51-45-43-27-2570-21,23130-29-23-24,0

771,39578-4865-4866-4867-49195-49199-49196-49200-52393-52392-49171-49172-156-157-47-53,23130-0-23-65281-10-11-35-16-5-13-18-51-45-43-27-43690-21,23130-29-23-24,0

771,2570-4865-4866-4867-49195-49199-49196-49200-52393-52392-49171-49172-156-157-47-53,27242-0-23-65281-10-11-35-16-5-13-18-51-45-43-27-51914-21,60138-29-23-24,0

771,31354-4865-4866-4867-49195-49199-49196-49200-52393-52392-49171-49172-156-157-47-53,10794-0-23-65281-10-11-35-16-5-13-18-51-45-43-27-19018-21,43690-29-23-24,0

771,51914-4865-4866-4867-49195-49199-49196-49200-52393-52392-49171-49172-156-157-47-53,23130-0-23-65281-10-11-35-16-5-13-18-51-45-43-27-64250-21,47802-29-23-24,0
```

As you can see, the JA3 is changing, because of the GREASE values (2570, 51914...). The JA3 fingerprint must stay unchanged (in the provided pcap) and the correct value is:

`771,4865-4866-4867-49195-49199-49196-49200-52393-52392-49171-49172-156-157-47-53,0-23-65281-10-11-35-16-5-13-18-51-45-43-27-21,29-23-24,0`
MD5: `b32309a26951912be7dba376398abc3b`

The pcap can be downloaded from `https://filesender.cesnet.cz/?s=download&token=d6bc86f7-2658-0cab-774a-2e3b963ccc2b`

Can you please add this pcap (using mergecap) into the `https-sample.pcap` and also rename it to `tls-sample.pcap
`
Thank you

**Review by havraji6 — APPROVED** [review]

**Pull request merged**